### DISCUSSION TITLE
Add search ability within the scope of single note

### DISCUSSION THREAD

**Discussion by MichalMichalak** [standardnotes/forum#3045] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Some of my notes are quite long and I found myself trying to press `CTRL/CMD+F` few times already to be able to find something in them. Nothing happens, there is no search box.

**Describe the solution you'd like**
Each note should have a search box available upon pressing above mentioned shortcut and user should be able to use it to look for strings of text. Not critical but it would be ideal to have a checkbox for whole words and regex search. Maybe search and replace, too.

**Describe alternatives you've considered**
I am not sure there is any. This seems to be very basic functionality present in most applications used to work with text.

**Additional context**
No screenshot, but just hit `CTRL/CMD+F` in VSCode, or in Notepad++, or many other text editors. Or in the web browser.
Thank you!

**Comment by avt00** [comment]
It's an ongoing issue and they've been ignoring this functionality for 3, maybe 5 years.

**Comment by MichalMichalak** [comment]
It's a bit sad that no one seems to be reading this forum. I don't expect devs to jump in and solve it as critical issue. Nevertheless it would be nice to, at least, acknowledge the problem.

**Comment by effieeee** [comment]
Hi all, while this feature isn't on Plain Text notes yet, we do have it on the **Super** note type, which is actively being developed 🙏

**Comment by MichalMichalak** [comment]
Thank you, that's good enough for me. Can't wait to see it rolling out to plain text and markdown. Meanwhile I'll go and check super, as I can see it in the app already. 🙇

**Comment by fra-gadio** [comment]
This is VERY important for me too.

**Comment by tweaksalot** [comment]
same here - it's sad that I have to copy the contents of a note into bbedit just to search for text within it. nvalt did this perfectly btw.
